website/docs/plugins/ray_launcher.md

```diff
@@ -0,0 +1,184 @@
+---
+id: ray_launcher
+title: Ray Launcher plugin
+sidebar_label: Ray Launcher plugin
+---
+<!-- Add PyPI links etc -->
+
+The Ray Launcher plugin provides 2 launchers: `ray_local` and `ray_aws`. `ray_local` will launch jobs locally on your laptop. `ray_aws` launches jobs remotely on AWS and is built on top of [Ray Autoscaler](https://docs.ray.io/en/latest/autoscaling.html).
+
+Install with
+
+```commandline
+pip install hydra_ray_launcher
```

Comment:
Until we have a package, let's keep the installation instructions honest:

`pip install plugins/hydra_ray_launcher`